.github/CHANGELOG.md

```diff
@@ -154,6 +154,53 @@ random_mat1 = rng.random((3,2))
 random_mat2 = rng.standard_normal(3, requires_grad=False)
 ```
 
+* PennyLane now has a ``kernels`` module.
+  It provides basic functionalities for computing quantum embedding kernels as well as 
```

Comment:
```suggestion
  It provides basic functionalities for working with quantum kernels as well as 
```
nit-picking: a) the computation itself is done by the user, so "working" may be better; b) the functionality should work for all quantum kernels, not only the "embedding" ones, right?